app/controllers/tools_controller.rb

```diff
@@ -8,7 +8,9 @@ class ToolsController < ApplicationController
   def index
     scope = Broadcast.current.published.order(publish_date: :desc, id: :desc)
     scope = scope.where(category: @category) if @category
-    @articles = scope.page(params[:page]).per(10)
+    @articles = scope.non_featured.page(params[:page]).per(10)
```

Comment:
To keep this more concise, `@articles` should order by featured, then by publish_date, then by id. The first three articles (if they are featured) are then rendered using the customized CSS (three at top), followed by any other articles.

Reply:
This would imply that featured articles are only displayed on the first page vs currently featured articles are displayed for each page followed by regular articles.
Do you intend to only show them on the first page? If so I can change the query.

Reply:
Correct, the featured articles would only show on page one. If you're navigating into further pages, you're searching for something that isn't featured to begin with, so it'd be displaying something the user has already decided to ignore.

Reply:
fair point